contentful/export.json

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
It looks like this is specific to your environment -- should we add this to `.gitignore`? Or remove it from this PR specifically?

Reply:
Taken from `https://github.com/contentful-userland/gatsby-contentful-starter/blob/master/contentful/export.json `. This json is used during the setup command to seed the space with all the example data (information/content/blog posts). 

Unfortunately if the user already created an example space, and uses that example spaceId as the spaceId here, ends up with duplicated blog posts. Bug is also existent in `https://github.com/contentful-userland/gatsby-contentful-starter` as well. Since there are multiple example spaces, I'm updating documentation to say that they should use an empty space.

Reply:
Ahh I see, I misunderstood the purpose of this file then.